apps/server/src/services/integration-service/OscIntegration.ts

```diff
@@ -100,7 +85,60 @@ export class OscIntegration implements IIntegration<OscSubscription> {
     this.oscClient.send(message);
   }
 
+  private initTX(enabledOut, targetIP, portOut, subscriptions) {
```

Comment:
would making the init functions public not help us restart during runtime?
we will likely want to restart the server and client separately 

Currently each private function is handling its own logic to verify the need for change, but this makes that the consumer needs to have access to the entire config instead of just the changes of the part (server / client) that should be restarted

(hope this made sense)

Reply:
hmmm maybe
we always get the whole OSCSettings struct from the client anyway

https://github.com/cpvalente/ontime/blob/22d1181443394d952b94a5268c4a0d6ec66e087d/apps/server/src/api-data/osc/osc.controller.ts#L13-L25


the consumer would then need to check what has changed...?